### PR TITLE
Upgrade packages and target frameworks

### PR DESCRIPTION
**Pull request by jakubch1**

### CODE REVIEW

File: CoveragePublisher.sln [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CoveragePublisher.Console", "src\CoveragePublisher.Console\CoveragePublisher.Console.csproj", "{D4A2E5B5-16AB-4FD5-BB64-9BE72A80B321}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{30652AFE-214B-4203-B0D1-AF2E358B49F6}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (vinayakmsft) [inline CoveragePublisher.sln:18]: How is this file updated? Is it auto generated?

Comment (jakubch1) [reply CoveragePublisher.sln:18]: This is updated by Visual Studio.

File: src/CoveragePublisher.Tests/ArgumentsProcessorTests.cs [hunk 2]
```diff
@@ -72,19 +72,6 @@ public void WillPrintHelpTextForInvalidArgs()
             Assert.IsTrue(Regex.Replace(helpText, @"\s+", "").Contains(UsageText), helpText);
         }
 
-        [DataTestMethod]
```
Comment (vinayakmsft) [inline src/CoveragePublisher.Tests/ArgumentsProcessorTests.cs:75]: Just to understand, why was this test method removed?

Comment (vinayakmsft) [reply src/CoveragePublisher.Tests/ArgumentsProcessorTests.cs:75]: ok got it

**Review by vinayakmsft — APPROVED** [review]

**Pull request merged**